### PR TITLE
RavenDB-12508 More detail about journals added to `StorageHandler`

### PR DESCRIPTION
**Pull request by haludi**
http://issues.hibernatingrhinos.com/issue/RavenDB-12508

### CODE REVIEW

File: src/Voron/Debugging/StorageReportGenerator.cs [hunk 1]
```diff
@@ -103,7 +106,14 @@ public DetailedStorageReport Generate(DetailedReportInput input)
                 tables.Add(tableReport);
             }
 
-            var journals = GenerateJournalsReport(input.Journals);
+            var journals = new JournalsReport
```
Comment (haludi) [inline src/Voron/Debugging/StorageReportGenerator.cs:109]: Maybe the collecting data should be under flush lock?

**Comment by ayende** [comment]
@haludi The code failed to compile, please fix this

File: src/Voron/Debugging/StorageReportGenerator.cs (hunk 1 above)
Comment (ayende) [reply src/Voron/Debugging/StorageReportGenerator.cs:109]: Doesn't actually matter, and might actually impact operations, so let's not do that

**Comment by arekpalinski** [comment]
@ml054 Probably we need to adjust Storage Report in the studio

**Review by ppekrol — APPROVED** [review]

**Pull request merged**